go.mod

```diff
@@ -1,38 +1,38 @@
 module github.com/grpc-ecosystem/grpcdebug
 
-go 1.22.7
-
-toolchain go1.23.6
+go 1.25.0
```

Comment:
Let's stick to 1.23 for now. That'll get rid of the toolchain line. I don't see any point of going newer in the go.mod.